File: Data Structures & Algorithms/max-consecutive-ones/submission-1.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        c=0
        m=0
        for i in range(len(nums)):
            if nums[i]==1:
                c+=1
            else:
                m=max(m,c)
                c=0
        m=max(m,c)
        return m
        

File: Data Structures & Algorithms/maximum-subarray/submission-1.java
class Solution {
    public int maxSubArray(int[] nums) {
        int sum=0;
        int max=Integer.MIN_VALUE;
        int i=0;
        while(i<nums.length){
            if(sum<0) sum=0;
            sum+=nums[i];
            max=Math.max(sum,max);
            i+=1;
        }
        return max;
    }
}
